db/hints/manager.cc

```diff
@@ -174,6 +174,9 @@ void manager::register_metrics(const sstring& group_name) {
         sm::make_counter("sent", _stats.sent,
                         sm::description("Number of sent hints.")),
 
+        sm::make_counter("sent_bytes_total", _stats.sent_hints_bytes_total,
```

Comment:
Why do the receive hints have skip_when_empty, and they sent don't?

Reply:
That's a good point. None of the hinted handoff metrics use `skip_when_empty`. I'm not knowledgeable in how exactly it's used (please, correct me if I'm wrong), but the way I understand it is the metrics won't be "displayed" unless they've been modified at least once.

In the context of hints, I think there's a merit in keeping them that way – when analysing the behaviour of a node, hints often play a significant role in how performant it is, so seeing no hints have been sent or written is useful because it gives you that information right away. The same reasoning could be applied to the metrics with incoming hints (which would mean I shouldn't have made them skipped when empty). But I'm sure it's still possible to obtain those metrics, so perhaps it'd be better to skip them when empty anyway. I definitely agree the metrics corresponding to errors, etc. should be skipped.